src/pages/PeopleTabPage.tsx

```diff
@@ -0,0 +1,54 @@
+import { useState, useEffect } from 'react';
+import { Loader } from '../components/Loader';
+import { PeopleTable } from '../components/PeopleTable';
+import { getPeople } from '../api';
+import { Person } from '../types';
+
+export const PeopleTabPage: React.FC = () => {
+  const [peopleFromServer, setPeopleFromServer] = useState<Person[]>([]);
+  const [isLoadin, setIsLoading] = useState(false);
+  const [errorMessage, setErrorMessage] = useState(false);
+
+  const isEmpty = peopleFromServer.length === 0 && !errorMessage;
+
+  useEffect(() => {
+    setIsLoading(true);
+    getPeople()
+      .then(setPeopleFromServer)
+      .catch(() => {
```

Comment:
In the `catch` block, you should also catch a possible `error` and return it to the user or output it to the console:
```
.catch((error) => setErrorMessage(`Something went wrong: ${error.message}`))
```
Then it will not be necessary to rename the name of the error state.